tests/gcp/operators/test_cloud_build.py

```diff
@@ -115,7 +115,7 @@ class TestGcpCloudBuildCreateBuildOperator(TestCase):
     @mock.patch("airflow.gcp.operators.cloud_build.CloudBuildHook")
```

Comment:
Can you change name of test class? TestGcpCloudBuildCreateBuildOperator => TestGcpCloudBuildCreateOperator

Reply:
sure! Thx.